docs/generating-a-provider.md

```diff
@@ -0,0 +1,391 @@
+# Generating a Crossplane Provider
+
+In this guide, we will generate a Crossplane provider based on an existing
+Terraform provider using Terrajet.
+
+We have chosen [Terraform GitHub provider] as an example, but the process will
+be quite similar for any other Terraform provider.
+
+## Generate
+
+1. Generate a GitHub repository for the Crossplane provider by hitting the
+   "**Use this template**" button in [provider-tf-template] repository.
```

Comment:
Is the repo `[provider-tf-template]` ready now? I'd like to go through it.

Reply:
Sorry I got the answer in the references https://github.com/crossplane-contrib/provider-tf-template. Thanks.